pkgs/tools/misc/crudini/default.nix

```diff
@@ -2,40 +2,47 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "crudini";
-  version = "0.9.3";
+  version = "0.9.4";
+  format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "pixelb";
     repo = "crudini";
     rev = version;
-    sha256 = "0298hvg0fpk0m0bjpwryj3icksbckwqqsr9w1ain55wf5s0v24k3";
+    hash = "sha256-jbTOaCF/ZqRpM0scDBBAcV5bSYg/QhBPbM9R5cONZ2o=";
   };
 
-  nativeBuildInputs = [ help2man installShellFiles ];
-
-  propagatedBuildInputs = with python3Packages; [ iniparse ];
-
   postPatch = ''
-    substituteInPlace crudini-help \
-      --replace ./crudini $out/bin/crudini
-    substituteInPlace tests/test.sh \
-      --replace ..: $out/bin:
+    patchShebangs crudini.py crudini-help tests/test.sh
+
+    # https://github.com/pixelb/crudini/issues/98
+    sed -i '/cat INSTALL/d' crudini-help
   '';
 
+  env.SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  nativeBuildInputs = [
+    help2man
+    installShellFiles
+    python3Packages.setuptools
+    python3Packages.setuptools-scm
+    python3Packages.wheel
+  ];
+
+  propagatedBuildInputs = with python3Packages; [ iniparse ];
+
   postInstall = ''
     # this just creates the man page
     make all
 
-    install -Dm444 -t $out/share/doc/${pname} README EXAMPLES
+    install -Dm444 -t $out/share/doc/${pname} README.md EXAMPLES
     installManPage *.1
   '';
 
   checkPhase = ''
     runHook preCheck
 
-    pushd tests >/dev/null
-    bash ./test.sh
-    popd >/dev/null
+    cd tests && ./test.sh
```

Comment:
we should do `&& cd ..` afterwards just in case any of the subsequent phases for whatever reason assumes as certain working dir.